extensions/wikia/ThemeDesigner/js/ThemeDesigner.js

```diff
@@ -576,9 +576,16 @@
 				reloadCSS = true;
 			}
 
-			if (setting === 'color-body' || setting === 'color-body-middle' || setting === 'color-page' ||
-				setting === 'color-buttons' || setting === 'color-links' || setting === 'background-image' ||
-				setting === 'color-header' || setting === 'wordmark-font') {
+			if (setting === 'color-body' ||
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Reduce the number of conditional operators (8) used in the expression (maximum allowed 3). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS1067)